meetballs/urls.py

```diff
@@ -30,4 +30,5 @@
     path('profile/<id>/', app_views.profile, name='profile'),
     path('edit-profile/', app_views.edit_profile, name='edit profile'),
 
-] + static(settings.STATIC_URL, document_root=settings.STATIC_ROOT)
+] + static(settings.STATIC_URL, document_root=settings.STATIC_ROOT)\
+              + static(settings.MEDIA_URL, document_root=settings.MEDIA_ROOT)
```

Comment:
I think this should work and will look cleaner.
```suggestion
urlpatterns = (
    [
        path("admin/", admin.site.urls),
        path("", app_views.home, name="home"),
        path("game-events/", include("game_event.urls")),
        path("login/", app_views.loginUser, name="loginUser"),
        path("logout/", app_views.logoutUser, name="logoutUser"),
        path("register/", app_views.registerUser, name="registerUser"),
        path("profile/<id>/", app_views.profile, name="profile"),
        path("edit-profile/", app_views.edit_profile, name="edit profile"),
    ]
    + static(settings.STATIC_URL, document_root=settings.STATIC_ROOT)
    + static(settings.MEDIA_URL, document_root=settings.MEDIA_ROOT)
)

```

Reply:
fixed